app/src/main/java/io/homeassistant/companion/android/util/vehicle/DomainCheck.kt

```diff
@@ -0,0 +1,19 @@
+package io.homeassistant.companion.android.util.vehicle
+
+import android.os.Build
+import androidx.annotation.RequiresApi
+import io.homeassistant.companion.android.common.data.integration.Entity
+import io.homeassistant.companion.android.common.data.integration.domain
+import io.homeassistant.companion.android.vehicle.MainVehicleScreen
+
+@RequiresApi(Build.VERSION_CODES.O)

```

Comment:
This annotation is incorrect, nothing in here requires O.

Reply:
The domain variable required it

Reply:
Hmm I think it may be because MainVehicleScreen is annotated? `domain` definitely isn't. Also see [this suggestion](https://github.com/home-assistant/android/pull/3732/files/5da46d30710dbf14354cbfe2c058bab6f99be25d#r1278292919) which could resolve it.

Reply:
Agreed, its now moved :)